src/main/java/core/basesyntax/FileCreator.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FileCreator {
+
+    public static void createFile() {
+        String fileName = "/Users/Test_user/IdeaProjects/jv-fruit-shop/reportToRead.csv";
+
+        try (PrintWriter writer = new PrintWriter(new BufferedWriter(new FileWriter(fileName)))) {
```

Comment:
Please use try-with-resources to ensure that resources are closed properly. This is good, but make sure all classes that work with I/O operations also follow this practice.